src/src/AppContext.js

```diff
@@ -14,6 +14,15 @@ function sleep(duration) {
   });
 }
 
+function truncateString(str) {
+  const maxLength = 70;
+  if (str.length > maxLength) {
+    return str.substring(0, maxLength) + "...";
+  } else {
+    return str;
+  }
+}
+
```

Comment:
can you put the as a parameter: you can put 70 as a default value.

Technically the string is longer than 70 characters, because of the ..., maybe use … instead? (yes i wrote a truncate function not long ago :D)

Reply:
Also should probably be in an utils file instead of part of the context